device.h

```diff
@@ -60,6 +60,9 @@ struct device_user {
 	struct list_head node;
 };
 
+#define device_has_control(_dev, _op) \
```

Comment:
This check macro is nice but I think the corresponding getter macro would be nice to make code calling ops simpler

Reply:
I have implemented a slightly different approach. Let me know what do you think about it.

Reply:
It’s much nicer !